x-pack/plugins/siem/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -89,63 +95,116 @@ export const PolicyList = React.memo(() => {
   const columns: Array<EuiTableFieldDataColumnType<Immutable<PolicyData>>> = useMemo(
     () => [
       {
-        field: 'name',
+        field: '',
         name: i18n.translate('xpack.siem.endpoint.policyList.nameField', {
           defaultMessage: 'Policy Name',
         }),
         // eslint-disable-next-line react/display-name
-        render: (value: string, item: Immutable<PolicyData>) => {
+        render: (item: Immutable<PolicyData>) => {
           const routePath = getManagementUrl({
             name: 'policyDetails',
             policyId: item.id,
             excludePrefix: true,
           });
           const routeUrl = getManagementUrl({ name: 'policyDetails', policyId: item.id });
-          return <PolicyLink name={value} route={routePath} href={routeUrl} />;
+          return (
+            <>
+              <EuiFlexItem grow={false}>
```

Comment:
I don't you can have `<EuiFlexItem>`'s outside of a `< EuiFlexGroup>`. Instead of the Fragment, just wrap it in `<EuiFlexGroup>`